### PR TITLE
Add an FAQ on how to debug/troubleshoot VPN connection issues on AirWatch

### PR DESCRIPTION
**Pull request by jasonblais**
Work in progress while we work through these steps with a customer. Hoping we can have this document referenceable with other customers in future who might hit the same connection issues.

### CODE REVIEW

File: source/mobile/mobile-troubleshoot.rst [hunk 1]
```diff
@@ -56,6 +56,26 @@ and then ignore the gradle task with
 
   ./gradlew assembleRelease -x bundleReleaseJsAndAssets
 
+I have mobile connection issues with per-app VPN deployed on AirWatch
+--------------------------------------------------------------------------------
+
+The following steps are for AirWatch, but similar steps can be taken to troubleshoot connection issues with per-app VPN deployed on other EMM providers, such as MobileIron or BlackBerry Dynamics.
```
[the remark this answers is not in the document]
Comment (jasonblais) [reply source/mobile/mobile-troubleshoot.rst]: @enahum Is the `IOS_BUILD_FOR_RELEASE` setting the same for all EMM providers?

[the remark this answers is not in the document]
Comment (enahum) [reply source/mobile/mobile-troubleshoot.rst]: yes, that setting is not for the EMM providers but for the build tool (fastlane)

**Comment by jasonblais** [comment]
Thanks @enahum - updated

**Comment by amyblais** [comment]
Do we know yet if this doc is needed?

**Comment by jasonblais** [comment]
@enahum I can also make this more generic, e.g. `How do I debug my mobile apps on an EMM provider`.

I think these steps are still useful to document. I'm realizing there isn't anything specific about debugging VPN in the steps.

**Comment by enahum** [comment]
I don’t really think this is needed, also this is more for developers than admins, so I really don’t know how useful it will be

**Comment by jasonblais** [comment]
Have closed since this wasn't deemed useful.

**Pull request closed without merging**